### ISSUE TITLE
Is there a way to skip a corpus test?

### ISSUE THREAD

**Issue by stsewd** [tree-sitter/tree-sitter#647]
Hi, I'm writing a grammar, and I've needed this a couple of times. Is there a way to skip just one test? I know there is the `--filter` option, but not really what I need.

**Comment by demizer** [comment]
A "jank" way I found to do this is to prepend two space characters to the test you want to skip.

```
  ==================
  Plain struct
  ==================
  
  struct Point {
      x: i64
      y: i64
  }
  
  ---
  
  (source_file)
```
The cli seems to skip this entire block. Not sure why this works, but it does.

**Comment by amaanq** [comment]
> I was thinking something more like a marker/metadata on each test.
> 
> ```
> ==================
> Test (skip)
> ==================
> 
> test with failures
> 
> ----
> 
> (some_nodes (that-are-not-working-yet)
> ```
> 
> If the test passes, it would be marked as an error in the `test` command.

Ideally, would the marker be fed into the cli arg? As in, something like `--skip-marker="(skip)"`? I don't think picking a universal skip marker would be good, as someone might just use that in their test name

**Comment by ahlinc** [comment]
I thought about different markers on my own needs like error nodes is expected in a tree (end prohibited by default).
I'd suggest the following syntax like:
```
==================
Test
:skip
:error
==================
```
So flags would be possible after a title and start with colon without any indents.

**Comment by ObserverOfTime** [comment]
Implemented in #3060.
